apps/cowswap-frontend/src/common/pure/CurrencySelectButton/index.tsx

```diff
@@ -69,7 +70,14 @@ export function CurrencySelectButton(props: CurrencySelectButtonProps): ReactNod
       displayTokenName={displayTokenName}
       displayChainName={displayChainName}
       role="button"
-      aria-label={currency ? `Selected token: ${currency.symbol}` : 'Select a token'}
+      aria-label={
+        currency
+          ? (() => {
+              const sym = currency.symbol || ''
+              return t`Selected token: ${sym}`
+            })()
```

Comment:
Nitpick: do it outside the tsx in a const, will be cleaner and not need the in place fn declaration and execution

Reply:
Done.